doc/specs/#182 - Support for installation of portable standalone apps.md

```diff
@@ -0,0 +1,196 @@
+
+No shortcuts or icons will be created for the package.
+
+>Note: Some portable programs may require dependencies. Dependencies are not covered in this specification and are not yet supported as of the writing of this specification.
+
+### Upgrade
+
+The package is upgraded in the same path as the installed version. The first step the Windows Package Manager will perform is an uninstall of the previous version of the package. If the file is in use, the user will be informed the package is running so they can shut it down. Optionally, the user may specify "--force" to forcefully shut the application down for upgrade.
+
+A new Windows Apps & Features entry is created to correctly report the upgraded version for future potential upgrades.
+
+There will be no support for installing multiple “side by side” versions of portable packages.
+
+### Uninstall
+
+The executable should be removed along with the entry in Apps & Features. The user should also be able to uninstall the package from Apps & Features.
+
+>Note: the default behavior for uninstalling a portable application with Windows Apps & Features will be to execute uninstall with "--purge" so any files created by the portable application will be removed if they are located in the portable applications directory.
+
+If the directory is empty after removing the portable application, the directory should also be deleted. If the directory is not empty, the user should be informed that other files exist in the directory so it will not be removed.
+
+### Manifest Validation
+
+Portable package manifests will only support zero or one "command" value to be specified. In the absence of the "--rename" argument, the value specified in "command" will define the default value for renaming the portable application and subsequently the value specified in the "AppPath" subkey.
+
+## UI/UX Design
+
+### Installing a portable package
+
+```text
+winget install Microsoft.NuGet
+Found NuGet [Microsoft.NuGet] Version 6.0
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://dist.nuget.org/win-x86-commandline/v6.0.0/nuget.exe
+  ██████████████████████████████   0.1 MB /  0.1 MB
+Successfully verified installer hash
+Starting package install...
+Successfully installed
+```
+
+### Upgrading a portable package
+
+```text
+winget upgrade Microsoft.NuGet
+Found NuGet [Microsoft.NuGet] Version 7.0
+This application is licensed to you by its owner.
+Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
+Downloading https://dist.nuget.org/win-x86-commandline/v7.0.0/nuget.exe
+  ██████████████████████████████   0.1 MB /  0.1 MB
+Successfully verified installer hash
+Starting package install...
+Successfully installed
+```
+
+If the portable application is running when the user performs an upgrade, the user will be informed and given options.
```

Comment:
"options" being actions that they could take, not "interactive command line questionnaire".

Reply:
Agree, I'm trying to ensure we avoid any user input. My intent with "given options" was just the additional text in the output with the hint for "--force".